metrics/exporter_test.go

```diff
@@ -101,23 +103,139 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func TestMetricsExporter(t *testing.T) {
-	_, err := newMetricsExporter(&metricsConfig{
-		domain:               servingDomain,
-		component:            testComponent,
-		backendDestination:   "unsupported",
-		stackdriverProjectID: ""}, TestLogger(t))
-	if err == nil {
-		t.Errorf("Expected an error for unsupported backend %v", err)
+var (
+	newMetricExporterTests = []struct {
```

Comment:
nit: Well actually I prefer not making this var global if this is only used for one test. I know this is how we did in `config_test.go`.